docs/first-time-run.md

```diff
@@ -119,7 +119,16 @@ when using this method, secret containing bot token should be created.
 oc create secret generic github-bot-token --from-literal github_bot_token=< BOT TOKEN >
 ```
 
-### Only Hosted pipeline:
+#### Hydra credentials
+To verify publishing checklist, Hosted pipeline uses Hydra API. To authenticate with
+Hydra over basic auth, secret containing service account credentials should be created.
+
+```bash
+oc create secret generic hydra-credentials \
```

Comment:
This secret should exists also in the Ansible

Reply:
Added